Could not find artifact org.eclipse.rdf4j:rdf4j-runtime:jar:3.7.4 in jitpack.io

When importing the library through Maven, the following error will occur upon executing a `mvn clean install` or `mvn -U test` command:

```
Error:  Failed to execute goal on project tapas-executor-3: Could not resolve dependencies for project ch.unisg.tapas:tapas-executor-3:jar:0.0.1-SNAPSHOT: Could not find artifact org.eclipse.rdf4j:rdf4j-runtime:jar:3.7.4 in jitpack.io (https://jitpack.io) -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Error: Process completed with exit code 1.
```

I believe this error is caused by this line which will use the specified Build repository for all dependencies.
https://github.com/Interactions-HSG/wot-td-java/blob/a8d5c864770a4289ed6e9cfec8428c9e8fc9fd7c/build.gradle#L54

However, I also tried using the Maven central repository to retrieve the runtime artifact. This will also result in this error despite the fact that the repository should contain the artifact according to [this](https://mvnrepository.com/artifact/org.eclipse.rdf4j/rdf4j-runtime):
```
[ERROR] Failed to execute goal on project tapas-executor-3: Could not resolve dependencies for project ch.unisg.tapas:tapas-executor-3:jar:0.0.1-SNAPSHOT: Could not find artifact org.eclipse.rdf4j:rdf4j-runtime:jar:3.7.4 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

To reproduce:
**pom.xml**
```xml
<repositories>
    <repository>
        <id>jitpack.io</id>
        <url>https://jitpack.io</url>
    </repository>
</repositories>
```
```xml
<dependency>
    <groupId>com.github.Interactions-HSG</groupId>
    <artifactId>wot-td-java</artifactId>
    <version>v0.1.0</version>
</dependency>
```
Run `mvn clean install`

@andreiciortea @

--- a/README.md
+++ b/README.md
@@ -55,7 +55,7 @@ Maven:
 
 Gradle:
 ```groovy
-implementation 'com.github.Interactions-HSG:wot-td-java:v0.1.0'
+implementation 'com.github.Interactions-HSG:wot-td-java:v0.1.1'
 ```
 
 Maven:
@@ -63,7 +63,7 @@ Maven:
 <dependency>
   <groupId>com.github.Interactions-HSG</groupId>
   <artifactId>wot-td-java</artifactId>
-  <version>v0.1.0</version>
+  <version>v0.1.1</version>
 </dependency>
 ```
 

--- a/build.gradle
+++ b/build.gradle
@@ -11,7 +11,7 @@ java {
   withSourcesJar()
 }
 
-version '0.1.0'
+version '0.1.1'
 
 repositories {
   jcenter()
@@ -48,7 +48,7 @@ publishing {
       from components.java
       groupId 'ch.unisg.ics.interactions'
       artifactId 'wot-td-java'
-      version '0.1.0'
+      version '0.1.1'
     }
   }
 

--- a/build.gradle
+++ b/build.gradle
@@ -11,7 +11,7 @@ java {
   withSourcesJar()
 }
 
-version '0.0.1'
+version '0.1.0'
 
 repositories {
   jcenter()
@@ -22,8 +22,11 @@ dependencies {
   implementation 'org.apache.httpcomponents.client5:httpclient5-fluent:5.1.2'
   implementation 'org.eclipse.californium:californium-core:2.6.3'
   implementation 'com.google.code.gson:gson:2.8.9'
-  implementation 'org.eclipse.rdf4j:rdf4j-runtime:3.7.4'
   implementation 'org.slf4j:slf4j-api:2.0.0-alpha5'
+
+  implementation('org.eclipse.rdf4j:rdf4j-runtime:3.7.4@pom') {
+    transitive = true;
+  }
 
   // Use JUnit test framework
   testImplementation 'junit:junit:4.13.2'
@@ -45,7 +48,7 @@ publishing {
       from components.java
       groupId 'ch.unisg.ics.interactions'
       artifactId 'wot-td-java'
-      version '0.0.1'
+      version '0.1.0'
     }
   }
 

